api/actions/__test__/createComment.spec.js

```diff
@@ -40,6 +50,8 @@ const reqWithoutWorkItem = {
 
 describe('createChatterComment action', () => {
     it('should call Issue.getName and Gus.createComment with workitem from title', async () => {
+        process.env.WORK_ITEM_BASE_URL =
+            'https://gus.lightning.force.com/lightning/r/ADM_Work__c/';
```

Comment:
Can we add a test without having an explit link to gus.lightning.force.com? git2gus needs to work with Agile Accelerator - which is a product on AppExchange

Reply:
I could set `process.env.WORK_ITEM_BASE_URL` to any value actually. The test is just verifying that `WORK_ITEM_BASE_URL` is used to create the expected URL. I was following pattern seen into `<root>/api/services/Issues/__test__/getWorkItemUrl.spec.js`